sound/soc/intel/boards/sof_sdw.c

```diff
@@ -1066,9 +1074,11 @@ static int sof_card_dai_links_create(struct device *dev,
 			      ssp_components, 1,
 			      NULL, info->ops);
 
-		ret = info->init(NULL, links + link_id, info, 0);
-		if (ret < 0)
-			return ret;
+		if (info->init) {
+			ret = info->init(NULL, links + link_id, info, 0);
+			if (ret < 0)
+				return ret;
+		}
```

Comment:
should be a separate patch.

Reply:
Ack.

Reply:
This is handled in latest revision, marking as resolved.